tests/memories_test.py

```diff
@@ -1389,7 +1394,6 @@ def f(inp):
     self.assertIn("input_output_alias", lowered_text)
     self.assertDeleted(x)
 
-  @jtu.run_on_devices('tpu')
```

Comment:
This test has nothing to do with compute offload. I don't know how it got here but I'll move it.